caddytest/integration/caddyfile_adapt/log_filter_with_header.txt

```diff
@@ -89,7 +89,10 @@ localhost {
 														{
 															"handle": [
 																{
-																	"access_logger_names": "health_check_log,general_log",
+																	"access_logger_names": [
+																		"health_check_log",
+																		"general_log"
+																	],
```

Comment:
Oh good, I wasn't sure if the `vars` handler would just take an array as-is. That's nice :+1: